packages/components/src/components/carousel/carousel.tsx

```diff
@@ -217,6 +218,7 @@ export class Carousel {
       breakpoints: getSplideBreakpoints(this.slidesPerPage as Exclude<BreakpointCustomizable<number> | 'auto', string>), // eslint-disable-line @typescript-eslint/no-redundant-type-constituents
       // https://splidejs.com/guides/i18n/#default-texts
       i18n: parseJSONAttribute(this.intl || {}), // can only be applied initially atm
+      direction: getLangDirection(this.host),
```

Comment:
Should this enable `rtl` support? Because still doesn't work

Reply:
What do you mean exactly by "it doesn't work"? This enables "rtl/ltr" support for the carousel which was not the case until now. The positioning of the items and the sliding direction should be now fixed. But yes, some e2e tests should comply the existing VRT's.

Reply:
I've added some basic e2e tests to verify rtl support of carousel.